authors/apps/articles/models.py

```diff
@@ -139,3 +139,12 @@ class Rating(models.Model):
                                 related_name='article_ratings',
                                 blank=True, null=True)
     user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+
+class Bookmark(models.Model):
```

Comment:
I think it was an interesting approach to go with a whole different model for this though, rather the more straightforward `bookmarked_by` approach and would like to understand that decision.